cassandra/src/main/java/com/yahoo/ycsb/db/CassandraCQLClient.java

```diff
@@ -195,6 +208,14 @@ public void init() throws DBException {
 
         session = cluster.connect(keyspace);
 
+        readStmts = new ConcurrentHashMap<Set<String>, PreparedStatement>();
```

Comment:
Since these don't rely on any information that's available at init time, could we move it into a static initializer? It'll simplify reasoning about them being used safely.

Or even just do object allocation where they're declared.

Reply:
Done. Allocated the objects as static initializers.